homework7/hw7task3/tests/test_tic_tac_toe_checker.py

```diff
@@ -0,0 +1,17 @@
+from typing import List
+
+import pytest
+from project3_.tic_tac_toe_checker import tic_tac_toe_checker
+
+
+@pytest.mark.parametrize(
+    ("game_board", "expected_result"),
+    [
+        ([["-", "-", "o"], ["-", "o", "o"], ["x", "x", "x"]], "x wins!"),
+        ([["o", "-", "o"], ["-", "o", "o"], ["x", "x", "o"]], "o wins!"),
+        ([["-", "-", "o"], ["-", "x", "o"], ["x", "o", "x"]], "unfinished!"),
+        ([["x", "o", "x"], ["o", "x", "o"], ["o", "x", "o"]], "draw!"),
+    ],
+)
+def test_tic_tac_toe_checker(game_board: List[List[str]], expected_result: str):
```

Comment:
I expect at least 6 tests for this task.  It will be much more convenient if you name your test cases.